src/app/market/sell/add-item/add-item.component.html

```diff
@@ -141,7 +141,7 @@
       <!-- TODO: add another input for fiat value and inter-connect the two: -->
       <mat-form-field class="full-width">
         <mat-icon matPrefix class="icon" fontSet="partIcon" fontIcon="part-cryptocurrency-particl"></mat-icon>
-        <input matInput min="0" placeholder="Price per item" formControlName="basePrice" type="number">
+        <input matInput min="0" (keypress)="inputValidate($event)" placeholder="Price per item *" formControlName="basePrice" type="number">
```

Comment:
This still leaves the option of copy/pasting values into the input. This can be mitigated by including a (paste) event handler (possibly pointing to the same method - this would require updating the validation method to include a check for the pasted data  (see comment with the example on the function itself).